### ISSUE TITLE
Cooldown timer

### ISSUE THREAD

**Issue by TheCrowbill** [ethereumclassic/tweets-etc_network#126]
Adding a cooldown timer to this action would allow for lengthy tweet queues yet prevent back-to-back tweets caused by overzealous reviewers.

**Comment by gitr0n1n** [comment]
Good idea. Only allows X number of tweets per day. Threaded tweet = 1 tweet.
